mssqlcli/main.py

```diff
@@ -63,7 +63,7 @@ def configure_and_update_options(options):
 
     if not options.integrated_auth:
         if not options.username:
-            options.username = input(u'Username (press enter for sa)') or u'sa'
+            options.username = input(u'Username (press enter for sa):') or u'sa'

```

Comment:
PR of the month right here! This has bugged me for so long :)

Reply:
My mistake is finally corrected